src/week-view.h

```diff
@@ -21,6 +21,9 @@
 #define FOCAL_TYPE_WEEK_VIEW (week_view_get_type())
 G_DECLARE_FINAL_TYPE(WeekView, week_view, FOCAL, WEEK_VIEW, GtkDrawingArea)
 
+void allocate_colors(void);
+void init_style(GtkWidget *mainWindow);
```

Comment:
Global functions should be prefixed with the module to which they relate `week_view_*`. But I think both these functions can be removed as I wrote above.